data/tags.json

```diff
@@ -54,38 +54,38 @@
       "more info on Matrix: <https://matrix.org/about/>, <https://matrix.org/bridges/>"
     ]
   },
-  { "name": "contr",
-    "aliases": [],
+  { "name": "contribute",
+    "aliases": ["contr", "contrib"],
```

Comment:
Could we please have "contributing" instead of "contr"? I don't think anyone's used or thought of "contr".